docs/cudf/source/io-supported-types.rst

```diff
@@ -0,0 +1,59 @@
+    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
+    | uint16                | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ❌     | ✅     | ❌     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     |
+    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
+    | uint32                | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ❌     | ✅     | ❌     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     |
+    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
+    | uint64                | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ❌     | ❌     | ❌     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     |
+    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
+    | float32               | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     |
+    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
+    | float64               | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     |
+    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
+    | bool                  | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     |
+    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
+    | str                   | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ✅     | ❌     | ❌     | ✅     | ✅     |
+    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
+    | category              | ✅     | ❌     | ❌     | ❌     | ✅     | ❌     | ✅     | ❌     | ❌     | ✅     | ✅     | ❌     | ❌     | ✅     | ✅     |
+    +-----------------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
+    | list                  | ❌     | ❌     | ✅     | ✅     | ✅     | ❌     | ❌     | ❌     | ❌     | ❌     | ❌     | ❌     | ❌     | ✅     | ✅     |
```

Comment:
Can we add `struct` and `decimal` rows as well?

Reply:
Added :+1: